scalability_and_performance/routing-optimization.adoc

```diff
@@ -9,4 +9,6 @@ The {product-title} HAProxy router scales to optimize performance.
 
 include::modules/baseline-router-performance.adoc[leveloffset=+1]
 
+For more information on Ingress sharding, see xref:../networking/ingress-operator.adoc#nw-ingress-sharding-route-labels[Configuring Ingress Controller sharding by using route labels] and xref:../networking/ingress-operator.adoc#nw-ingress-sharding-namespace-labels[Configuring Ingress Controller sharding by using namespace labels].
```

Comment:
The xrefs are landing at the top of the page for me. Perhaps the tags need `_configuring-ingress` on the end? 
`xref:../networking/ingress-operator.adoc#nw-ingress-sharding-route-labels_configuring-ingress`
`xref:../networking/ingress-operator.adoc#nw-ingress-sharding-namespace-labels_configuring-ingress`